kv/db.go

```diff
@@ -146,3 +147,78 @@ func (s *DBServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", contentType)
 	w.Write(body)
 }
+
+// RegisterRPC ...
+func (s *DBServer) RegisterRPC(rpcServer *rpc.Server) error {
+	return rpcServer.RegisterName("Server", &s.rpcDBServer)
+}
+
+type rpcDBServer struct {
```

Comment:
Yep, just for creating a separate method namespace. I like the multiraft approach better. Changed.